src/views/about/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+/**
+ * About View
+ *
+ * This component should have everything to be rendered
+ * under /about route. It is intended to have a description
+ * about the data and what we're doing with the app
```

Comment:
nit: We generally try to avoid using "we" in comments. I would also recommend just saying what this component does rather than the intention. Maybe rephrase to something like "It contains a description of the data and the purpose of the app."

Reply:
Awesome, I readapted almost every comment to avoid "we" and start with a verb